src/controls/Jalousie.ts

```diff
@@ -41,6 +64,30 @@ export class Jalousie extends ControlBase {
                 // TODO: re-add: smartIgnore: false
             },
         );
+        this.addStateEventHandler(
```

Comment:
Please move this call inside the `if (control.states.position) {` below.
I don't remember the reason for that "if", but since it is there, we must assume there are situations where `control.states.position` is not set.

Reply:
OK.